plugins/exec.lua

```diff
@@ -11,6 +11,32 @@ local function exec(cmd, keep_newline)
 end
 
 
+local function exec_rw(cmd, keep_newline, str)
+  local proc = process.start { "sh", "-c", cmd }
```

Comment:
I wonder if it would be more appropriate to either just execute the command without `sh`, or to add things like `cmd` for Windows.

Reply:
Thank you for your review.

If the shell is removed then all the features provided by a shell (like piping) will be lost, so IMHO the benefits of calling the shell totally outweights the costs.

About supporting Windows, AFAIK currently there're PowerShell and the "old" `cmd` (I don't know if ~command.com~ is still shipped), so a deeper investigation will be necessary.